config/backstop/engine_scripts/playwright/overrideCSS.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * OVERRIDE CSS
+ * Apply this CSS to the loaded page, as a way to override styles.
+ *
+ * Use this in an onReady script E.G.
+  ```
+  module.exports = async function(page, scenario) {
+    await require('./overrideCSS')(page, scenario);
+  }
```

Comment:
Remove?

Reply:
this is default/examples that backstop provides, ill leave this commented for now but any commenting ive done of my code ill remove